documentation/specs/Banner.md

```diff
@@ -0,0 +1,46 @@
+# Banner Component Specification
+
+## Overview
+
+The `Banner` component displays a prominent message above the navigation bar at the top of the page that cannot be dismissed.
+
+### Use Cases
+
+The primary use case of the `Banner` component is to inform the user of promotions and other offerings on the marketing (logged out) site. **Not** to be used to call attention to the user as a result of interacting with the UI.
+
+---
+
+## Design
+
+### API
+
+```typescript
+export type BannerColor = "primary" | "success" | "neutral";
+export type BannerProps = {
+  /**
+   * Banner color
+   * @default "primary"
+   */
+  color?: BannerColor;
```

Comment:
**question:** is there a more semantic name we could use instead of `color`? something that represents more of the state of the banner rather than an effect of the state.

i notice Shopify uses `status` in a similar vein, MUI uses `severity` for its `Alert`, Twilio uses `variant` for its `Callout`, etc

Reply:
i went ahead and changed it to `status`